scripts/mdpow-pcw

```diff
@@ -184,29 +190,40 @@ if __name__ == "__main__":
     parser.add_option('-p', '--plotfile', dest="plotfile",
                       help="plot dV/dlambda to FILE; use png or pdf suffix to "
                       "determine the file type. 'auto' generates a pdf file "
-                      "DIRECTORY/dVdl_MOLID_pcw.pdf and 'None' disables it [%default]",
+                      "DIRECTORY/dVdl_MOLID_pow.pdf and 'None' disables it [%default]",
                       metavar="FILE")
     parser.add_option('-o', '--outfile', dest="outfile",
                       help="append one-line results summary to FILE [%default]",
                       metavar="FILE")
     parser.add_option('-e', '--energies', dest="energyfile",
                       help="append solvation free energies to FILE [%default]",
                       metavar="FILE")
+    parser.add_option('--estimator', dest="estimator", default='mdpow',
+                      help="choose the estimator to be used, alchemlyb or mdpow estimators")
+    parser.add_option('--method', dest="method", default='TI',
+                      help="choose the method to calculate free energy",)
     parser.add_option('--force', dest='force',
                       action="store_true",
                       help="force rereading all data [%default]")
+    parser.add_option('--SI', dest='SI',
+                      action="store_true",
+                      help="Apply statistical inefficiency analysis")
```

Comment:
Is SI enabled by default? Should be yes when MBAR is the default so that we automatically get good error estimates.

Reply:
If MBAR and SI is the default then Is the sentence under stride still correct, i.e., is it true with SI+MBAR that stride>1 has little impact on precision?? If in doubt, remove the sentence so that we don't provide wrong advice.

The default for stride should probably be 1 when used with SI.

Reply:
Stride should be 1 in all cases. The user can increase manually this value if the analysis is too slow, being aware of consequences.

stride>1 has an important impact on precision in many cases. stride=10 was an arbitrary default value, useful with TI, which now should be replaced with the result of SI.